### PR TITLE
Add Pumpkin AI plating kit

### PR DESCRIPTION
**Pull request by JORJ949**
## About the PR 
Adds a new pumpkin AI skin and plating kit
image
Add wiring to a pumpkin to get the new plating kit!

## Why's this needed? 
Funny halloween themed AI skin.

## Changelog 

```changelog
(u)JORJ949
(*)Added a Pumpkin AI plating kit. Add wires to a pumpkin to get it!
```

### CODE REVIEW

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]
Oh I love this, only concern is that the recipe is maybe too simple and will lead to confusion among botanists trying to make a jack-O-lantern. Maybe make it adding wires or metal sheets to a pumpkin instead, so it's more explicit and easy to remember what you're making?
Also maaaybe the recipe should be Spooktober/Autumn ifdefed?

**Comment by JORJ949** [comment]
> Also maaaybe the recipe should be Spooktober/Autumn ifdefed?

None of the other pumpkin stuff is locked behind halloween so I don't think this should be.

File: code/modules/food_and_drink/plants.dm [hunk 1]
```diff
@@ -1074,8 +1074,14 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 	brew_result = list("juice_pumpkin"= 40)
 	var/obj/item/clothing/head/carving_result = /obj/item/clothing/head/pumpkin
 	var/obj/item/reagent_containers/food/spoon_result = /obj/item/reagent_containers/food/drinks/bowl/pumpkin
+	var/obj/item/ai_plating_kit/snip_result = /obj/item/ai_plating_kit/pumpkin
```
Comment (TobleroneSwordfish) [inline code/modules/food_and_drink/plants.dm:1077]: Name no longer accurate

File: code/modules/food_and_drink/plants.dm [hunk 2]
```diff
@@ -1074,8 +1074,14 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
...
 
 	attackby(obj/item/W, mob/user)
+		if (istype(W, /obj/item/cable_coil))
+			user.visible_message(SPAN_NOTICE("[user] adds wiring to the [src]!"))
```
Comment (TobleroneSwordfish) [inline code/modules/food_and_drink/plants.dm:1081]: should take some `amount` from the cable coil with `change_stack_amount()`

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Pull request merged**